test/functional/test_framework/test_framework.py

```diff
@@ -347,9 +347,15 @@ def join_network(self):
         connect_nodes_bi(self.nodes, 1, 2)
         self.sync_all()
 
+    def sync_blocks(self, nodes=None, **kwargs):
+        sync_blocks(nodes or self.nodes, **kwargs)
+
+    def sync_mempools(self, nodes=None, **kwargs):
+        sync_mempools(nodes or self.nodes, **kwargs)
+
     def sync_all(self, nodes=None, **kwargs):
-            sync_blocks(nodes or self.nodes, **kwargs)
-            sync_mempools(nodes or self.nodes, wait=0.1, wait_func=lambda: self.bump_mocktime(3, True), **kwargs)
+        self.sync_blocks(nodes, **kwargs)
+        self.sync_mempools(nodes or self.nodes, wait=0.1, wait_func=lambda: self.bump_mocktime(3, True), **kwargs)
```

Comment:
Why is this sync mempool call different https://github.com/bitcoin/bitcoin/pull/15773/commits/fa6dc7c5c3477bda04481f080d430f63539974e2#diff-64721c5ee64d44f7114d6d0d2226db4dR417?

Should it not be... 
```suggestion
        self.sync_mempools(nodes, wait=0.1, wait_func=lambda: self.bump_mocktime(3, True), **kwargs)
```

Reply:
Yepp, you're right. Fixed it.